ipatests/test_webui/test_automember.py

```diff
@@ -323,7 +332,7 @@ def test_rebuild_membership_hosts(self):
         self.delete_host_groups('webservers')
 
     @screenshot
-    def test_rebuild_membership_users(self):
+    def test_rebuild_membership_users(self, confirm_btn="ok"):
```

Comment:
No need to add confirm_btn as a parameter of the test function (the function is never called with a different value for confirm_btn).